skeleton/local-deploy/deployment/README.md

```diff
@@ -0,0 +1,78 @@
+# Deploying Locally
+
+This document is intended for users who wish to deploy their application locally. This was generated from a software template and information regarding image building and deployment are found below. 
+
+## Quick Links
+
+- [Prerequisites](#prerequisite)
+
+- [Building and Pushing Images](#building-and-pushing-image-manually)
+
+- [Deployment](#deploying-to-podman-desktop-ai-lab)
+
+- [Cleanup](#cleanup)
+
+- [Troubleshooting](#troubleshooting)
+
+## Prerequisite
+
+Deploying locally via Podman Desktop is done with the assistance of the `AI Lab Extension`. Please ensure you have the following downloaded and installed:
+- [Podman Desktop 1.8.0+](https://github.com/containers/podman-desktop)
+- [Podman 4.9.0+](https://podman.io/)
+- [Podman Desktop AI Extension](https://github.com/containers/podman-desktop-extension-ai-lab?tab=readme-ov-file#installation)
+
+## Building and Pushing Image Manually
+
+You have chosen `${{ values.imageRegistry }}` as your desired image registry. Please ensure you are logged in and can read/write images from that repository. For example using `quay.io` you can run this command to prompt the entering of credentials:
+```
+podman login quay.io
+```
+
+To rebuild images you can run the following command from the root of the repository:
+```
+bash deployment/build_and_push.sh <absolute path to docker/containerfile>
```

Comment:
`ai-lab.yaml` specifies two images that need to be built: the application Containerfile, and the llamacpp Containerfile, but the script only builds one image. The user needs to override the IMAGE_FULL variable to push the llamacpp image. The instructions should probably cover building both images, since it's not immediately clear which images need to be built and where.

What if we point the script to the user's repo, and it finds the necessary Containerfiles to build, based on known locations?

Reply:
I'm wondering now if the build script is even required since Podman Desktop will build the images for you if they don't exist, wdyt?

Reply:
IMO, it's nice to have as an option. Maybe move the manual build instructions below the deploy like I suggest in https://github.com/redhat-ai-dev/ai-lab-template-experiment/pull/1#discussion_r1671203988